libpod/oci_conmon_common.go

```diff
@@ -1114,8 +1114,16 @@ func (r *ConmonOCIRuntime) createOCIContainer(ctr *Container, restoreOptions *Co
 		args = append(args, []string{"--exit-command-arg", arg}...)
 	}
 
-	// Pass down the LISTEN_* environment (see #10443).
 	preserveFDs := ctr.config.PreserveFDs
+	preserveFDsMap := make(map[uint]interface{})
```

Comment:
should use struct{} over interface{} as it doesn't need to store any info.
But maybe do we even have to use a map here at all?
Given you already iterate anyway I think this would be simpler:
```go
files := make([]*os.File, len(ctr.config.PreserveFDsList))
for _, i := range ctr.config.PreserveFDsList {
		if i > preserveFDs {
			preserveFDs = i
		}
		f = os.NewFile()
        files[i] = f
}
cmd.ExtraFiles = files
```